config/sites/presidentialsearch.uiowa.edu/config_split.patch.user.role.webmaster.yml

```diff
@@ -0,0 +1,23 @@
+adding: {  }
+removing:
+  permissions:
+    - 'access any webform configuration'
```

Comment:
Are these permission removals intentional? I think webform might be but can't quite remember. Panopto perms were maybe just never updated.

Reply:
Removed panopto perms.

Reply:
Seems like it: https://github.com/uiowa/uiowa/pull/2679